modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -46,11 +61,43 @@ $ oc patch Network.config.openshift.io cluster \
   --type='merge' --patch '{ "spec": { "networkType": "OVNKubernetes" } }'
 ----
 
-. To confirm the migration disabled the OpenShift SDN default CNI network provider and removed all OpenShift SDN Pods, enter the following command. It might take several moments for all the OpenShift SDN Pods to stop.
+. Optional: You can customize the maximum transmission unit (MTU) and the port used by the Geneve (Generic Network Virtualization Encapsulation) overlay network to meet your network infrastructure requirements. Enter the following command to open the Cluster Network Operator (CNO) configuration in an editor:
```

Comment:
@pliurh, this adds an optional step for configuring the CNO with a different MTU or Geneve port.

Reply:
Users can also change the clusterNetwork CIDR during the migration. But  the serviceNetwork shall not be changed.

Reply:
Changing the Network.config should be put in one step.

Reply:
@pliurh, okay, that makes sense.